kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -168,43 +168,48 @@
           </KGridItem>
         </KGrid>
 
-        <AccordionContainer :items="activeQuestions">
-          <template #left-actions>
-            <KCheckbox
-              ref="selectAllCheckbox"
-              class="select-all-box"
-              :label="selectAllLabel$()"
-              :checked="allQuestionsSelected"
-              :indeterminate="selectAllIsIndeterminate"
-              @change="() => selectAllQuestions()"
-            />
-          </template>
-          <template #right-actions>
-            <KIconButton
-              icon="expandAll"
-              :tooltip="expandAll$()"
-              :disabled="!canExpandAll"
-              @click="expandAll"
-            />
-            <KIconButton
-              icon="collapseAll"
-              :tooltip="collapseAll$()"
-              :disabled="!canCollapseAll"
-              @click="collapseAll"
-            />
-            <KIconButton
-              icon="refresh"
-              :tooltip="replaceAction$()"
-              :disabled="!canReplaceQuestions"
-              @click="handleReplaceSelection()"
-            />
-            <KIconButton
-              icon="trash"
-              :tooltip="coreString('deleteAction')"
-              :aria-label="coreString('deleteAction')"
-              :disabled="selectedActiveQuestions.length === 0"
-              @click="() => deleteQuestions()"
-            />
+        <AccordionContainer>
+          <template #header="{ canExpandAll, expandAll, canCollapseAll, collapseAll }">
+            <div style="display: flex; align-items: center; justify-content: space-between">
```

Comment:
All these inline styles should be updated in the future, for now I have used them to quick prototype how the new api would look like